modules/optimization/src/optimizationreporters/OptimizationReporter.C

```diff
@@ -41,7 +42,16 @@ OptimizationReporter::validParams()
 }
 
 OptimizationReporter::OptimizationReporter(const InputParameters & parameters)
-  : OptimizationReporterBase(parameters)
+  : OptimizationReporterBase(parameters),
+    _opt_data(parameters),
+    _measurement_values(_opt_data.getMeasurementValues()),
+    _simulation_values(_opt_data.getSimValues()),
+    _misfit_values(_opt_data.getMisfitValues())
+{
+  setICsandBounds();
```

Comment:
This the only derived class of OptimizationReporterBase that implements this method should it should be moved to this class.

Reply:
This is the only change that really needs to be made.